lib/firmware/fsm_msg_tendermint.h

```diff
@@ -0,0 +1,232 @@
+
+void fsm_msgTendermintGetAddress(const TendermintGetAddress *msg) {
+  RESP_INIT(TendermintAddress);
+
+  CHECK_INITIALIZED
+
+  CHECK_PIN
+
+  const CoinType *coin = fsm_getCoin(true, msg->chain_name);
+  if (!coin) {
+    return;
+  }
+  HDNode *node = fsm_getDerivedNode(SECP256K1_NAME, msg->address_n,
+                                    msg->address_n_count, NULL);
+  if (!node) {
+    return;
+  }
+
+  hdnode_fill_public_key(node);
+
+  if (!tendermint_getAddress(node, msg->chain_name, resp->address)) {
+    memzero(node, sizeof(*node));
```

Comment:
Is it worthwhile in the interest of maintaining consistent style to casting structures to `void *` type in call to `memzero()` as on line 145 below?

Reply:
fixed